.github/workflows/test_build.yml

```diff
@@ -95,7 +95,7 @@ jobs:
           sudo apt-get install -y -q --no-install-recommends libatomic-ops-dev libboost-stacktrace-dev \
                   libncurses-dev libncurses5-dev libreadline-dev libeigen3-dev liblapack-dev libxml2-dev \
                   libgc-dev libgdbm-dev libglpk-dev libgmp3-dev libgtest-dev libmpfr-dev libntl-dev gfan \
-                  libcdd-dev libgivaro-dev fflas-ffpack w3c-markup-validator
+                  libcdd-dev libgivaro-dev libboost-regex-dev fflas-ffpack w3c-markup-validator
```

Comment:
The need for a new library, such as this one, should be documented in the file "INSTALL" for each OS, with the appropriate name there, and the configure script should verify its presence.

(Why didn't I need to install such a thing on Fedora?  Maybe it's just part of libboost.)

Reply:
There's no need, it always comes with boost.

Reply:
Also, the configure script does check for it thanks to @d-torrance.

Reply:
> There's no need, it always comes with boost.

If that were true, you wouldn't have to add it to the workflow.

Also, I don't see "libboost-regex-dev" anywhere in the "INSTALL" file.

Reply:
> Also, I don't see "libboost-regex-dev" anywhere in the "INSTALL" file.

I just checked, `libboost-regex-dev` is installed as a recommended package along with `libboost`. That's why it's not mentioned. But sure, I'll add it in the regex PR.

> If that were true, you wouldn't have to add it to the workflow.

Notice that libboost is not installed on the workflow.

Reply:
> Also, the configure script does check for it thanks to @d-torrance.

Oh, good.

Reply:
I've just checked: under Fedora, no new library is needed, for `/usr/lib64/libboost_regex.so` is included in `boost-devel`, which we already call for in `INSTALL`.